storage/client/queue/README.md

```diff
@@ -184,22 +184,22 @@ The following sections provide several code snippets covering some of the most c
 - [Set/Update a Queue metadata](#Set-a-queue-metadata)
 
 ### Build a client
-We have two ways of building QueueService or Queue Client. Here will take queueServiceClient as an example. Same things apply to queueClient.
+We have two ways of building QueueStorage or Queue Client. Here will take queueStorageClient as an example. Same things apply to queueClient.
```

Comment:
Where does QueueClient come into play in these examples?

Reply:
Do you think QueueClient need the code example here?
I have one sentence as description.

>  Here will take QueueStorageClient as an example. Same things apply to QueueClient.